### PR TITLE
Functions for creating raster layer and map

### PR DESCRIPTION
**Pull request by JohnRFoster**
Changed code to have functions instead of hard coding the raster and maps

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<<<<<<< HEAD
+.Rproj.user
+.Rhistory
+.RData
+.Ruserdata
+=======
```
Comment (mdietze) [inline .gitignore]: This shows a merge conflict, which needs to be resolved before the code can be pulled

File: Citizen Science API.R [hunk 2]
```diff
@@ -0,0 +1,172 @@
+library(rgbif)
```
Comment (mdietze) [inline Citizen Science API.R]: It appears that you changed the file name to switch _ to spaces. Please change back. First, it's bad practice to include spaces in file names. Second, the change in name makes it impossible to see the difference between the previous version and the new version -- git things these are completely different files.

**Review by mdietze — CHANGES_REQUESTED** [review]
There's two binary files that have been added, GADM_2.8_USA_adm0.rds and GADM_2.8_USA_adm1.rds, but they aren't referenced anywhere in the file so it's not clear how/why they're used. Also, in general it's not good practice to include inputs/outputs in a git repo -- they bloat the repository and version control can't manage their differences over time easily. If these files are NOT required, you'll need to do some non-trivial advanced git work to remove them in a way that doesn't retain them in the repo history (which is needed to prevent bloat) -- touch base with folks in the lab (especially Alexey) for help.

Also not clear why there's a NEFI_tick subfolder added within what everyone else will view as a NEFI_tick folder.

**Comment by JohnRFoster** [comment]
Cleaned up repo, and changed script name

**Review by mdietze — COMMENTED** [review]
script name still not back to what it was originally. Still merge conflicts in gitignore

File: CaryMouse_MetOut_code.R [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+     lambda.Recruit,
+     n.mean.Recruit,
+     #n.caught,
+     effect.size.Recruit,
+     plot.Recruit,
+     trace.Recruit,
+     ci.N.Recruit,
+     file = "/projectnb/dietzelab/fosterj/CaryMouseRecruit.RData")
+
+prin("END FILE")
```
Comment (mdietze) [inline CaryMouse_MetOut_code.R]: Is this supposed to be `print`

File: CaryTickMet.R [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+                    "beta.r")
+
+#xx <- as.numeric(Sys.getenv("SGE_TASK_ID")) # read array job number to paste into output file
+
+jags.out <- coda.samples(jags.model,
+                         n.iter = 100,
+                         variable.names = variable.names)
+
+saveRDS(jags.out, 
+        file = paste("/projectnb/dietzelab/fosterj/Previous_Runs_Tick/CaryTickNull_NegBinomLarv_", xx, ".rds",
```
Comment (mdietze) [inline CaryTickMet.R]: general suggestion: Define folder names near the top of any file/function and then construct file paths by using the `file.path` function to combine that folder with the specific file name. This makes the code much more portable

File: CaryTick_MonthlyMet.R [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+             R = R,
+             time = ncol(dat.t),
+             yr_mon.index = yr_mon.index,
+             temp = temp,
+             precip = precip,
+             rh = rh,
+             b0 = as.vector(c(0, 0, 0)),   # regression beta means
+             Vb = solve(diag(10000,3)))   # regression beta precisions
+
+noise <- function(dat, sd){
```
Comment (mdietze) [inline CaryTick_MonthlyMet.R]: There's too much in the PR for me to follow everything, but I feel like there is a **LOT** of redundant code in this repo. If you're reusing functions, define them once in their own files. If you're writing the same code again and again (or worse, copy-and-pasting) then you should figure out what parts are repeated and put them in functions. For example, I think a lot of your files contain code that is identical except for small differences in JAGS models -- to me that seems like something better handled by having one bit of code that handles the I/O for a specific class of models and then pass different individual models to that code.

File: CaryTick_miceTwo.R [hunk 6]
```diff
@@ -0,0 +1,244 @@
+library(rjags)
```
Comment (mdietze) [inline CaryTick_miceTwo.R]: These files have no headers explaining what the different models are.  This example, where there's a `CaryTick_miceOne.R` and a `CaryTick_miceTwo.R` is particularly bad as there's no one other than you who would have any idea what these different models are. And I guarantee that in a few months you won't know what they are either.

File: Cary_pop/CaryTick [hunk 7]
```diff
@@ -0,0 +1,448 @@
+"","Year","Grid","Date","Larvae.m2","Nymphs.m2","Adults.m2","X","day.of.year"
```
Comment (mdietze) [inline Cary_pop/CaryTick:1]: Do you have permission to make the Cary data public? If not, then these data files should not be in the PR

File: Cary_pop/Met Data/DAILY1995.csv [hunk 8]
```diff
@@ -0,0 +1,366 @@
+DATE,AVE_TEMP,AVE_RH,TOT_PREC,MIN_TEMP,MIN_TEMT,MIN_RH,MIN_RHT,MAX_WS,MAX_WST,MAX_TEMP,MAX_TEMT,MAX_RH,MAX_RHT,MAX_GSW,MAX_GSWT,MAX_DSW,MAX_DSWT,MAX_NETR,MAX_NETT,MAX_GPAR,MAX_GPAT,MAX_DPAR,MAX_DPAT,TOT_GSW,TOT_DSW,TOT_GPAR,TOT_DPAR,POT_PAR,POT_SW,KT_PAR,KT_SW,MIN_NETR,TOT_NETR,AVE_BP,AVE_SWS,AVE_RWD,AVE_RWS,WD_SD
```
Comment (mdietze) [inline Cary_pop/Met Data/DAILY1995.csv:1]: Please don't use spaces in folder names

**Review by mdietze — CHANGES_REQUESTED** [review]

**Pull request closed without merging**